benches/grpc/main.rs

```diff
@@ -0,0 +1,26 @@
+mod grpc_client;
+mod grpc_server;
+
+use criterion::{criterion_group, criterion_main, Criterion};
+use grpc_client::gcli;
+use grpc_server::server;
+
+fn grpc_benchmark(c: &mut Criterion) {
+    std::thread::spawn(|| {
```

Comment:
Should the server and client run in different cores?

Reply:
nice suggestion